snapcraft/project/_project_options.py

```diff
@@ -231,6 +231,10 @@ def __init__(
         self._set_machine(target_deb_arch)
         self.__debug = debug
 
+        # These paths maintain backwards compatibility.
+        self.internal_dir = os.path.join(self.__project_dir, "snap", ".snapcraft")
```

Comment:
maybe these two need to be _ to indicate they should not be used from plugins

Reply:
Agreed.

Reply:
yeah, I can move it to project